geospaas_harvesting/harvest.py

```diff
@@ -227,7 +238,12 @@ def main():
                             or not previous_result):
                         #Start a new process
                         results[harvester_name] = pool.apply_async(
-                            launch_harvest, (harvester_name, harvester_config))
+                            launch_harvest, (
+                                harvester_name,
+                                harvester_config,
+                                config.get('dump_on_interruption', True)
```

Comment:
With the True in this line, you set the default value of it as True if there is no declaration of it in the harvest.yml file?

Reply:
yes